controllers/allArticles-controller.js

```diff
@@ -0,0 +1,17 @@
+// REQUIRE
+const {fetchAllArticles} = require("../models/all-articles-model");
+
+// FUNCTION
+function getAllArticles(req, res, next) {
+    fetchAllArticles()
+      .then((articles) => {
+        return res.status(200).send(articles);
+      })
```

Comment:
Please make sure you return your data on an appropriate key e.g. {articles: articles}